snafu-derive/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![allow(unknown_lints, bare_trait_objects)]
 #![recursion_limit = "128"] // https://github.com/rust-lang/rust/issues/62059
+#![cfg_attr(feature = "std-backtraces", feature(backtrace))]
```

Comment:
Why is this needed? I would have thought it isn't, as we are only generating code that would use this feature, not using it ourselves.